week-3/F-array-forEach/exercise.js

```diff
@@ -8,6 +8,17 @@
 */
 
 var arr = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15];
+arr.forEach(function(number) {
+  if (number % 3 === 0 && number % 5 === 0) {
```

Comment:
Really nicely done! 👏 FizzBuzz is a famous exercise for testing your programming skill 🙂